src/alert/alert.ts

```diff
@@ -23,43 +25,61 @@ import {NgbAlertConfig} from './alert-config';
   selector: 'ngb-alert',
   changeDetection: ChangeDetectionStrategy.OnPush,
   encapsulation: ViewEncapsulation.None,
-  host: {'role': 'alert', 'class': 'alert', '[class.alert-dismissible]': 'dismissible'},
+  host:
+      {'role': 'alert', 'class': 'alert show', '[class.alert-dismissible]': 'dismissible', '[class.fade]': 'animation'},
   template: `
     <ng-content></ng-content>
     <button *ngIf="dismissible" type="button" class="close" aria-label="Close" i18n-aria-label="@@ngb.alert.close"
-      (click)="closeHandler()">
+      (click)="close()">
       <span aria-hidden="true">&times;</span>
     </button>
     `,
   styleUrls: ['./alert.scss']
 })
 export class NgbAlert implements OnInit,
     OnChanges {
+  /**
+   * If `true`, alert closing will be animated.
+   *
+   * Animation is triggered only when clicked on the close button (×)
+   * or via the `.close()` function
+   */
+  @Input() animation: boolean;
+
   /**
    * If `true`, alert can be dismissed by the user.
    *
    * The close button (×) will be displayed and you can be notified
    * of the event with the `(close)` output.
    */
   @Input() dismissible: boolean;
+
   /**
    * Type of the alert.
    *
    * Bootstrap provides styles for the following types: `'success'`, `'info'`, `'warning'`, `'danger'`, `'primary'`,
    * `'secondary'`, `'light'` and `'dark'`.
    */
   @Input() type: string;
+
   /**
    * An event emitted when the close button is clicked. It has no payload and only relevant for dismissible alerts.
    */
-  @Output() close = new EventEmitter<void>();
+  @Output('close') closeEvent = new EventEmitter<void>();
```

Comment:
Also I suggest we avoid doing this, because it is different in the template and in the code and it might be confusing:

```html
<ngb-alert (close)="...">
```
vs 
```ts
alert.closeEvent.subscribe(...)
```

As we're breaking API anyway, maybe we should do this:

```html
<ngb-alert #a #ngIf="closed" (closed)="closed = true"></ngb-alert> ←note 'closed', not 'close'
<button (click)="a.close()">Close</button>
```

It will be aligned with bootstrap's event name too

Reply:
I am fine with that

Reply:
Changing to `(closed)` then